pkg/golinters/recv/testdata/recv.go

```diff
@@ -0,0 +1,22 @@
+//golangcitest:args -Erecv
+package testdata
+
+import "fmt"
+
+type Foo struct { // want `the methods of "Foo" use different receiver names: f, fo.`
+	Name string
+}
+
+func (f Foo) A()  {}
+func (fo Foo) B() {}
+
+type Bar struct{} // want `the methods of "Bar" use pointer receiver and non pointer receiver.`
```

Comment:
```suggestion
type Bar struct{} // want `the methods of "Bar" use pointer receiver and non-pointer receiver.`
```

if you agree with https://github.com/ldez/recv/pull/1